wildvvad/utils/helper_functions.py

```diff
@@ -0,0 +1,115 @@
+"""
+This callback is inspired by the CSVLogger callback from Keras.
+reference:
+    https://keras.io/api/callbacks/csv_logger/
+"""
+import collections
+import csv
+import time
+import numpy as np
+import tensorflow as tf
+keras = tf.keras
```

Comment:
You can do it like this but normally you would do something like from tensorflow import keras